Microsoft.Recognizers.Text.DateTime/English/Parsers/EnglishDateTimePeriodParserConfiguration.cs

```diff
@@ -69,26 +72,29 @@ public bool GetMatchedTimeRange(string text, out string timeStr, out int beginHo
             beginHour = 0;
             endHour = 0;
             endMin = 0;
-            if (trimedText.EndsWith("morning"))
+            if (trimedText.EndsWith("morning") || trimedText.StartsWith("morning"))
```

Comment:
Don't repeat strings. Declare a constant and re-use it. Or create a regex if more are likely to be added later.